### PR TITLE
hide_funder_details_2273 done

### PR DESCRIPTION
**Pull request by GatteJyoti**
##### Checklist

- [ ] linter status: 100% pass
- [ ] changes don't break existing behavior
- [ ] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes

- #2273

### CODE REVIEW

**Comment by Anish-Agnihotri** [comment]
Hey, 

Thanks for picking up on #2273! Seems like you just need to do a quick lint for this PR to pass all the checks. 

Changing `if (result.bounty_owner_name == "Anonymous")` to `if (result.bounty_owner_name == 'Anonymous')` and running `make fix-eslint` on the file should fix the issues!

Thanks again.

Anish

**Comment by mbeacom** [comment]
:star: PR built and available in a preview environment **gitcoinco-web-pr-2389** [here](http://web.jx-gitcoinco-web-pr-2389.ci.gitcoin.co)

File: app/dashboard/templates/bounty/details.html [hunk 1]
```diff
@@ -172,11 +172,11 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
                     <span class="bounty-text" id="bounty_owner_address"></span>
                   </div>
                   <div>
-                    <span class="bounty-subheading">{% trans "Email" %} :</span>
+                    <span class="bounty-subheading" id="bounty_owner_email_div">{% trans "Email" %} :</span>
```
Comment (thelostone-mc) [inline app/dashboard/templates/bounty/details.html]: rename to `bounty_owner_label` ?

File: app/dashboard/templates/bounty/details.html [hunk 2]
```diff
@@ -172,11 +172,11 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
...
                     <span class="bounty-text" id="bounty_owner_email"></span>
                   </div>
                   <div>
-                    <span class="bounty-subheading">{% trans "Gitcoin Profile" %} :</span>
+                    <span class="bounty-subheading" id="bounty_owner_github_username_div">{% trans "Gitcoin Profile" %} :</span>
```
Comment (thelostone-mc) [inline app/dashboard/templates/bounty/details.html]: rename to `bounty_owner_github_username _label` ?

**Comment by thelostone-mc** [comment]
Closing this out as it's been moved over to https://github.com/gitcoinco/web/pull/2464
Thanks @GatteJyoti for your contribution :)

**Pull request closed without merging**